content/en/blog/_posts/2023-03-08-memory-qos-cgroups-v2/index.md

```diff
@@ -0,0 +1,275 @@
+---
+layout: blog
+title: 'Alpha Support for Quality-of-Service for Memory Resources'
+date: 2023-03-08
+slug: qos-memory-resources
+---
+
+**Authors:** Dixita Narang (Google)
+
+Kubernetes v1.27, released in April 2023, introduced changes to an alpha feature,
```

Comment:
The change itself is about a new Alpha feature rather than an existing one?

```suggestion
Kubernetes v1.27, released in April 2023, introduced changes as an alpha feature,
```

Reply:
These are changes to an existing alpha feature